pkg/util/execdetails/execdetails.go

```diff
@@ -206,8 +205,11 @@ func (d ExecDetails) String() string {
 	if d.BackoffTime > 0 {
 		parts = append(parts, BackoffTimeStr+": "+strconv.FormatFloat(d.BackoffTime.Seconds(), 'f', -1, 64))
 	}
-	if d.LockKeysDuration > 0 {
-		parts = append(parts, LockKeysTimeStr+": "+strconv.FormatFloat(d.LockKeysDuration.Seconds(), 'f', -1, 64))
+	lockKeyDetails := d.LockKeysDetail
+	if lockKeyDetails != nil {
+		if lockKeyDetails.TotalTime > 0 {
+			parts = append(parts, LockKeysTimeStr+": "+strconv.FormatFloat(lockKeyDetails.TotalTime.Seconds(), 'f', -1, 64))
```

Comment:
Better to have a case to cover the `lockKeyDetails` exists in the slow log.

Reply:
Will add ut for this case

Reply:
Modify the TestString test case in execdetails_test.go to cover LockKeys_time in SlowLog.